app/client/sass/_navigation-tabs.scss

```diff
@@ -23,4 +23,7 @@
       }
     }
   }
+  .nav-tab-a{
```

Comment:
what's the significance of the a? if it's just to avoid conflicts with other classes, just call something like sscs-nav-tab